tests/execute/result/check_results/test.fmf

```diff
@@ -0,0 +1,84 @@
+summary: Tests for check results behaviour
+description: Verify that check results are correctly interpreted and affect test results
+
+/check-pass:
+    summary: Test with passing checks
+    test: echo "Test passed"
+    framework: shell
+    duration: 1m
+    check:
+        - how: dmesg
+          result: respect
+    # Expected outcome: PASS (test passes, check passes)
```

Comment:
I would say this content is an excellent candidate for `description`. With `tmt show` in the test directory you would quickly learn all necessary info.

Reply:
Done in https://github.com/teemtee/tmt/pull/3239/commits/e873871d7fd0bfd9294e3f7ce1daa31a1e232491  
I've also used `enabled` instead of commenting-out the 'multiple' test. Make sense?

Reply:
Yeah, `enabled` definitely makes sense here. Thanks!